test/gnosisSafePersonalEditionCreation.js

```diff
@@ -161,5 +165,28 @@ contract('GnosisSafePersonalEdition', function(accounts) {
         let creationData = await getCreationData(token.address, 1337)
         await deployWithCreationData(creationData)
         assert.equal(await web3.eth.getCode(creationData.safe), '0x0')
-    })
+
+        const method = Web3EthAbi.encodeFunctionSignature('transfer(address,uint256)');
```

Comment:
to we still need `safeUtils.deployToken` or is it possible to clean it up (since we perform all token tests with the mock)

Reply:
I think safeUtils.deployToken is no longer required as we have 3 tests covering token transfer failures.